community/sample-apps/vuejs-auth0-graphql/app/README.md

```diff
@@ -102,7 +102,7 @@ To do this, open `auth_config.json`, and replace the values within with your own
 ```
 
 ## Create the initial tables
-1. Add your database URL and admin secret in `hasura/config.yaml`
+1. Add your graphql URL (like https://your-app.herokuapp.com/v1/graphql) and admin secret in `hasura/config.yaml`
```

Comment:
Here the endpoint will not have /v1/graphql.
It should be (https://your-app.herokuapp.com), since the CLI takes care of applying the migrations with the right endpoint.